packages/composables/src/useUser/index.ts

```diff
@@ -35,10 +38,13 @@ const params: UseUserFactoryParams<User, any, any> = {
 
   register: async (context: Context, { email, password, firstName, lastName }) => {
     try {
-      const guestCartToken = context.cart?.value?.token;
+      const guestCartToken = context.useCart.cart?.value?.token;
       await context.$spree.api.registerUser({ email, password, firstName, lastName });
       await context.$spree.api.logIn({ username: email, password, guestCartToken });
 
+      const wishlist = await context.$spree.api.getWishlist();
```

Comment:
What will getWishlist return if wishlists are not enabled? An empty placeholder?